buildbot/configure.py

```diff
@@ -135,7 +135,8 @@ def do_configure(args):
         # For clang-format, clang-tidy and code coverage
         llvm_enable_projects += ";clang-tools-extra;compiler-rt"
         # Build with zstd enabled on CI.
-        llvm_enable_zstd = "ON"
+        # Disable in sycl-web , we don't have zstd in pulldown env.
```

Comment:
sycl-web is merged to sycl, so we better have the same behavior in both branches.

Reply:
this is to unblock the pulldown first , I expect that @uditagarwal97 will have a better fix to make sure enabling zstd on a machine without zstd build as well.

Even if we can't get the good fix in time before next pulldown to sycl branch, we can resolve the conflicts to keep it on for sycl branch.

Reply:
@jsji To have the same behavior in sycl and sycl-web, we can disable zstd even in sycl branch for now. I'm investigating the build issue and will re-enable zstd after fixing.

Reply:
> @jsji To have the same behavior in sycl and sycl-web, we can disable zstd even in sycl branch for now. I'm investigating the build issue and will re-enable zstd after fixing.

This plan sounds good me.